src/autocomplete/style.js

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from 'styled-components';
+
+const DarkBg = css`
+  background-color: #f0f0f8;
+`;
+
+export const Container = styled.div``;
+
+export const SuggestionList = styled.ul`
+  background-color: #fff;
+  list-style-type: none;
+  width: min-content;
```

Comment:
I don't think this is good idea to include in an OSS library if it's not fully supported in the major browsers.

https://caniuse.com/#search=min-content